### PR TITLE
Grammar improvements & update link

### PR DESCRIPTION
**Pull request by kek-mex**

### CODE REVIEW

**Comment by kek-mex** [comment]
@blrhc

File: README.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ Currently, all bounties will be made `active`, funded, and will be evaluated by
 
 The payouts will be made in [monero](https://web.getmonero.org/) unless noted otherwise. Our choice of using monero as the method of payment is that it's both a well established and reputable project, and arguably has better privacy features than some of the other options.
 
-Jsgenesis is also making weekly payouts for participation on the [Joystream testnets](http://testnet.joystream.org/). More information about the what you can, what you can make, and why we're doing this can be found in our [helpdesk repo](https://github.com/Joystream/helpdesk).
+Jsgenesis is also making weekly payouts for participation on the [Joystream testnets](https://testnet.joystream.org/). More information about what you can make, and why we're doing this can be found in our [helpdesk repo](https://github.com/Joystream/helpdesk).
```
Comment (kek-mex) [inline README.md]: Maybe this should be changed to "More information about why we're doing this, and how you can participate can be found in our [helpdesk repo](https://github.com/Joystream/helpdesk)." ?

Comment (blrhc) [reply README.md:41]: Excellent suggestion!

Comment (blrhc) [reply README.md:41]: If you could commit that one, I'll merge right away. Apologies for the delay in reviewing this.

**Pull request merged**